yarn-error.log

```diff
@@ -0,0 +1,6202 @@
+Arguments: 
```

Comment:
this file doesn't need to be committed. Delete this. Also, can you add this to .gitignore?